tests/cluster/delta/test_delta_class.py

```diff
@@ -68,6 +70,15 @@ def test_02_write(self):
         dh.append(df, mergeSchema=True)
 
     def test_03_create(self):
+        print(Configurator().get_all_details())
```

Comment:
see the output in the raw logs:

2022-11-02T12:01:32.7658657Z ...........{'ID': '', 'MNT': 'mnt', 'resourceName': 'githubatc33772648181', 'storageAccount': 'abfss://silver@githubatc33772648181.dfs.core.windows.net', 'MyDb': 'TestDb', 'MyDb_name': 'TestDb', 'MyDb_path': 'abfss://silver@githubatc33772648181.dfs.core.windows.net/testdb', 'MyTbl': 'TestDb.TestTbl', 'MyTbl_name': 'TestDb.TestTbl', 'MyTbl_path': 'abfss://silver@githubatc33772648181.dfs.core.windows.net/testdb/testtbl', 'MyTbl2': 'TestDb.TestTbl2', 'MyTbl2_name': 'TestDb.TestTbl2', 'MyTbl3_path': 'abfss://silver@githubatc33772648181.dfs.core.windows.net/testdb/testtbl3'}
2022-11-02T12:01:32.7662760Z {'fs.azure.account.auth.type.githubatc33772648181.dfs.core.windows.net': 'OAuth', 'fs.azure.account.oauth2.client.secret.githubatc33772648181.dfs.core.windows.net': 'iTVXXXdyK', 'fs.azure.account.oauth.provider.type.githubatc33772648181.dfs.core.windows.net': 'org.apache.hadoop.fs.azurebfs.oauth2.ClientCreokenProvider', 'fs.azure.account.oauth2.client.endpoint.githubatc33772648181.dfs.core.windows.net': 'https://login.microsoftonline.com/1ddXXX/oauth2/token', 'fs.azure.account.oauth2.client.id.githubatc33772648181.dfs.core.windows.net': 'e24eXXXXd6'}

The XXX are by me. I don't see what is wrong here, and I don't see anything different to what we do in our working reference system. 

Anyone with a good idea, pleas pitch in.